thoughts.txt

```diff
@@ -0,0 +1,79 @@
+Options:
```

Comment:
💭 Maybe we should move this to docs or a GH pages

Reply:
whoops yep didn't realize I committed our notes lol